content/support-matrix/delfin/huawei/oceanstor/_index.md

```diff
@@ -0,0 +1,524 @@
+---
+title: OceanStor
+menuTitle: OceanStor
+description: "OceanStor Support Matrix"
+weight: 20
+disableToc: false
+tags: ["support matrix", "delfin", "drivers", "huawei"]
+---
+
+### Model and versions supported
+
+<table>
+   <thead>
+      <tr>
+         <th>Models</th>
+         <th>Management Software Versions</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>Storage Management Software</td>
```

Comment:
This does not seem correct. Please check the values @Shruthi-1MN @NajmudheenCT

Reply:
It is correct

Reply:
@Shruthi-1MN Under the `Models` column It says `Storage Management Software`. Shouldn't this be a list of Oceanstor models?